prov/usnic/src/fi_ext_usnic.h

```diff
@@ -104,13 +104,22 @@ struct fi_usnic_info_v2 {
 	struct fi_usnic_cap	**ui_caps;
 };
 
+/* In API version 1.2 and below, the v1 structure did not contain any 64-bit
+ * data types and therefore had a 4-byte alignment. Once v2 of the extension API
+ * was introduced in version 1.3, the extra pointers mandated an 8-byte
+ * alignment thus changing the offset of the v1 structure. This means that
+ * the post-1.3 getinfo extension API is not compatible with the previous
+ * versions (unless patched). Make the alignment explicit and consistent by
```

Comment:
Replace the "This means that..." sentence with similar language from the man page:

> This means that the alignment difference manifests when an application using v1 of the extension is compiled Libfabric v1.1.x or v1.2.x, but then runs with `libfabric.so` that is v1.3.x or higher.